modern/src/resources/l10n/en.json

```diff
@@ -177,6 +177,8 @@
     "loginReset": "Reset Password",
     "loginRegister": "Register",
     "loginLogin": "Login",
+    "loginLoginOIDC": "Login with SSO",
+    "loginOIDCRedirect": "Authenticating with SSO...",
```

Comment:
Maybe just use a progress bar or something to indicate redirect? We try to avoid adding new string when possible.